src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/GuardArrow.java

```diff
@@ -61,7 +67,7 @@ protected void arrowHit(final EntityLivingBase targetEntity)
     {
         super.arrowHit(targetEntity);
         Log.getLogger().info("Arrow hit " + targetEntity + " with " + targetEntity.getHealth());
-        if (targetEntity.getHealth() <= 0.0F)
+        if (targetEntity.getHealth() <= 0.0F && !(shootingEntity instanceof EntityArcherBarbarian))
```

Comment:
nahhh, it's to stop it crashing, the stuff after only applies if the shooter of the arrow was a guard, otherwise it crashes (because it tries to get the shooters colony, which a barbarian doesn't have a colony..